src/pygambit/game.pxi

```diff
@@ -672,16 +672,20 @@ class Game:
                     mspr[s] = Rational(v)
             return mspr
 
-    def mixed_behavior_profile(self, rational=False) -> MixedBehaviorProfile:
-        """Create a behavior strategy profile over the game.
-
-        The profile is initialized to uniform randomization for each player
-        over their actions at each information set.
+    def mixed_behavior_profile(self, data=None, rational=False) -> MixedBehaviorProfile:
+        """Returns a behavior strategy profile `MixedBehaviorProfile` over the game,
```

Comment:
I think it's useful as a convention to try to keep the first sentence of the docstring concise, as that's what the user will see first in the documentation.  Then after a blank line, we can be more verbose about the operation of the function.  This is the general pattern numpy/scipy/pandas &c follow.